jib-core/src/main/java/com/google/cloud/tools/jib/cache/LayerEntriesSelector.java

```diff
@@ -56,11 +60,16 @@
 
     private final String sourceFile;
     private final String extractionPath;
+    private final Instant lastModifiedTime;
 
     @VisibleForTesting
-    LayerEntryTemplate(LayerEntry layerEntry) {
+    LayerEntryTemplate(LayerEntry layerEntry) throws IOException {
       sourceFile = layerEntry.getAbsoluteSourceFileString();
       extractionPath = layerEntry.getAbsoluteExtractionPathString();
+      lastModifiedTime =
+          Files.exists(layerEntry.getSourceFile())
```

Comment:
Hmm, I don't think any of `LayerEntry` files should not exist. This should probably just let `Files.getLastModifiedTime` throw its exception.